classes/controller/ProductListingFrontController.php

```diff
@@ -388,6 +390,40 @@ protected function getProductSearchVariables()
         return $searchVariables;
     }
 
+    /**
+     * @param ProductSearchResult $result
```

Comment:
Please describe the purpose of this method in the Phpdoc header.

Reply:
described and renamed